WORKSPACE

```diff
@@ -2,35 +2,36 @@ workspace(name = "cilium")
 
 register_toolchains("//bazel/toolchains:all")
 
+ENVOY_PROJECT = "envoyproxy"
+
+ENVOY_REPO = "envoy"
+
+# Envoy GIT commit SHA of release
 #
 # We grep for the following line to generate SOURCE_VERSION file for non-git
 # distribution builds. This line must start with the string ENVOY_SHA followed by
 # an equals sign and a git SHA in double quotes.
 #
 # No other line in this file may have ENVOY_SHA followed by an equals sign!
 #
-ENVOY_PROJECT = "envoyproxy"
-
-ENVOY_REPO = "envoy"
-
-# https://github.com/envoyproxy/envoy/tree/v1.25.5
-# NOTE: Update version number to file 'ENVOY_VERSION' to keep test and build docker images
-# for different versions.
+# renovate: datasource=github-releases depName=envoyproxy/envoy digestVersion=v1.25.5
 ENVOY_SHA = "4b5f5474f35763423684c0fe25c99cc7b2a01fcf"
 
-ENVOY_SHA256 = "00191a1e702040ec8f92ba18344b001e862e55cc6e24a60509db450bdeb8c0ee"
-
 # // clang-format off: unexpected @bazel_tools reference, please indirect via a definition in //bazel
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```

Comment:
http_archive was used as in the past envoy maintainer might release from the forked repo, but I didn't see it anymore for the last few releases, so all good.

Reply:
@sayboras great - thanks for the explanation!